hiring/interviewing.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Interviewing @ Artsy"
+description: An overview of our interview process
+---
+
+## What to expect from an Artsy interview
+
+We're excited that you've decided to interview for a job at Artsy! This document will cover our standard interview
+process. Our goal is to give you the information you need to feel prepared.
+
+We'll update this as often as we can when things change, but note that the details may shift slightly. Our
+Recruitment team will be available to answer any questions you may have throughout the hiring process!
+
+This guide is tailored to our standard interviews for the IC3-IC5 levels on
+[our ladder](/careers/ladder.md#readme). The content of the interviews may change slightly for other levels, but
+the ethos will be the same.
```

Comment:
Listing a short summary about the 3 phases can be helpful to get a big picture of the entire process.